irods/src/main/java/ch/cyberduck/core/irods/IRODSSession.java

```diff
@@ -63,7 +63,8 @@
 public class IRODSSession extends SSLSession<IRODSConnection> {
 
     static {
-        IRODSApi.setApplicationName(new PreferencesUseragentProvider().get());
+        // TODO: Use "new PreferencesUseragentProvider().get()" once irods/irods#8733 is resolved.
+        IRODSApi.setApplicationName("Cyberduck");
```

Comment:
Please use 

```suggestion
        IRODSApi.setApplicationName(PreferencesFactory.get().getProperty("application.name"));
```

as the library is not limited to usages in Cyberduck.